spec/unit/puppetlabs_spec_helper/tasks/beaker_spec.rb

```diff
@@ -25,5 +25,21 @@
       allow(ENV).to receive(:[]).and_return('"high", "medium", "low"')
       expect { described_class.setup_beaker(task) }.to raise_error(RuntimeError, %r{not a valid test tier})
     end
+    it 'errors when tiers are not in the allowe list' do
+      allow(ENV).to receive(:[]).and_return('foobar')
+      expect { described_class.setup_beaker(task) }.to raise_error(RuntimeError, %r{not a valid test tier})
+    end
+    it 'Override TEST_TIERS_ALLOWED' do
+      ENV['TEST_TIERS_ALLOWED'] = 'dev,rnd'
```

Comment:
Please set up mocking like in the next line, instead of altering the global environment.

Background: using the `allow(ENV).to receive(:[]).with('TEST_TIERS').and_return('dev')` syntax means that everything is cleaned up in all cases. Modifying global state means that when errors happen, cleanup is skipped, leading to flaky tests.